sample/prebid-config.yaml

```diff
@@ -12,6 +12,11 @@ adapters:
     enabled: true
 metrics:
   prefix: prebid
+analytics:
+  log:
+    enabled: true
+  greenbids:
+    enabled: true
```

Comment:
since it's a config sample and used for the very first start-up I suggest disabling greenbids features